### PR TITLE
E_CLASSROOM-344 [Bugfix] Dimensions of cards in dashboard should be fixed

### PR DESCRIPTION
**Pull request by Teraaay**
### Issue Link

- https://framgiaph.backlog.com/view/E_CLASSROOM-344

### Definition of Done
* [x] Recently taken quiz cards should have a fix width
* [x] Category learned and friend's activities should have a fixed height

#### Pages Affected
- Dashboard Page: http://localhost:3003/

### Scenarios/ Test Cases
- [x] If there is only one recent quiz card, it should not occupy the remaining spaces.
- [x] If there no categories learned or friend activities, the card height should not shrink.

### Screenshots
image
image
image

### CODE REVIEW

**Comment by catampongan** [comment]
## Approved PR #171 
- [x] Layout is fixed, height does not shrink
- [x] If only one recent card quiz, the layout is not broken
-----------------------------
image
image
image

-------------------------
- Total Test Time: 0.25 hr
- Total Iterations: 0
- Total Comments: 0

File: src/pages/student/main/Dashboard/index.js [hunk 1]
```diff
@@ -1,20 +1,19 @@
...
+import style from './index.module.scss';
 
 function Dashboard() {
-  const [categorieslearned, setCategoriesLearned] = useState(null);
   const userId = Cookies.get('user_id');
-  const [recentQuizzes, setRecentQuizzes] = useState(null);
+
   const [quizzes, setQuizzes] = useState(null);
+  const [recentQuizzes, setRecentQuizzes] = useState(null);
+  const [categorieslearned, setCategoriesLearned] = useState(null);
```
Comment (karlolee50) [inline src/pages/student/main/Dashboard/index.js]: use camel case
```suggestion
  const [categoriesLearned, setCategoriesLearned] = useState(null);
```

File: src/pages/student/main/Dashboard/index.js [hunk 2]
```diff
@@ -35,38 +34,33 @@ function Dashboard() {
...
-          </Card.Body>
-        </Card>
-      </Col>
+      <Card className={style.card}>
+        <Card.Header className={style.cardHeader}>
+          <p className={style.cardTitle}>Categories Learned</p>
+        </Card.Header>
+        <Card.Body>
+          {categorieslearned?.length ? (
+            categorieslearned?.map((categorylearned, idx) => {
```
Comment (karlolee50) [inline src/pages/student/main/Dashboard/index.js]: update here as well

File: src/pages/student/main/Dashboard/index.js [hunk 3]
```diff
@@ -35,38 +34,33 @@ function Dashboard() {
...
+              return (
+                <CategoryLearned key={idx} categorylearned={categorylearned} />
```
Comment (karlolee50) [inline src/pages/student/main/Dashboard/index.js]: if you can, please also update the prop of CategoryLearned component

**Review by karlolee50 — CHANGES_REQUESTED** [review]
Review time: 10 minutes
Comment/s: 2

**Review by karlolee50 — APPROVED** [review]

**Pull request merged**